fury/ui.py

```diff
@@ -3270,17 +3284,22 @@ def __init__(self, labels, padding=1, font_size=18,
             Absolute coordinates (x, y) of the lower-left corner of
             the button of the first option.
         """
+        if len(checked_labels) > 1:
+            raise ValueError("Only one option \
```

Comment:
I suggest using a temporary variable instead of breaking the text with a `\`. e.g.
```python
if len(checked_labels) > 1:
    msg = "Only one option can be pre-selected for radio buttons."
    raise ValueError(msg)
```
If needed you can break text string like so
```python
msg = ("Only one option can be pre-selected"
       " for radio buttons.")
```